[docs] Migrate sorting (#233)
mui-org/material-ui-x: docs/src/pages/components/data-grid/accessibility/accessibility.md

|                  <kbd>Space</kbd> | Navigate to the next scrollable page          |
|                <kbd>Page Up</kbd> | Navigate to the next scrollable page          |
|              <kbd>Page Down</kbd> | Navigate to the previous scrollable page      |

### Selection

|                                                           Keys | Description                                       |
| -------------------------------------------------------------: | :------------------------------------------------ |
|                            <kbd>Shift</kbd> + <kbd>Space</kbd> | Select the current row                            |
| <kbd>Shift</kbd> + <kbd>Space</kbd> + <kbd>Arrow Up/Down</kbd> | Select the current row and the row above or below |
|                                 <kbd>CTRL</kbd> + <kbd>A</kbd> | Select all rows                                   |
|                                 <kbd>CTRL</kbd> + <kbd>C</kbd> | Copy the currently selected row                   |

### Sorting

|                    Keys | Description          |
| ----------------------: | :------------------- |
| <kbd>CTRL</kbd> + Click | Enable multi-sorting |

### Key assignment conventions

> The above key assignments are defined in the context of Windows and Linux.

Reviewer: ```suggestion
The above key assignments are for Windows and Linux.
```
